src/components/can_cooperation/reverse_sdl_pt.json

```diff
@@ -2439,8 +2382,8 @@
                 "default_hmi": "NONE",
                 "moduleType": ["RADIO"],
                 "groups": ["Base-4"],
-                "groups_primaryRC": ["Base-4", "Base-RC", "Radio", "Climate", "RemoteControl"],
-                "groups_nonPrimaryRC": ["Base-RC", "Radio", "Climate", "Notifications-RC", "RemoteControl"],
```

Comment:
Wasn't OnPermissionChange in Notification-RC? How will passenger's device's apps receive these notifications?